frontend/src/components/resourceMap/nodes/KubeObjectNode.tsx

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
Can you please add justifications for disabling eslint and typescript things next to the disable lines?

Also, please check other changes in this PR if that's necessary too?

Reply:
removed it, wasn't needed anymore